src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlServerTransparentDataEncryptionCertificate.md

```diff
@@ -0,0 +1,201 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/Add-AzureRmSqlServerTransparentDataEncryptionCertificate
+schema: 2.0.0
+---
+
+# Add-AzureRmSqlServerTransparentDataEncryptionCertificate
+
+## SYNOPSIS
+Adds a Transparent Data Encryption Certificate for the given SQL Server instance
+
+## SYNTAX
+
+### AddAzureRmSqlServerTransparentDataEncryptionCertificate Input Object Parameter Set
+```
+Add-AzureRmSqlServerTransparentDataEncryptionCertificate [-InputObject] <AzureSqlServerModel>
+ [-PrivateBlob] <String> [-Password] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### AddAzureRmSqlServerTransparentDataEncryptionCertificate Resource Id Parameter Set
+```
+Add-AzureRmSqlServerTransparentDataEncryptionCertificate [-ResourceId] <String> [-PrivateBlob] <String>
+ [-Password] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### AddAzureRmSqlServerTransparentDataEncryptionCertificate Default Parameter Set
+```
+Add-AzureRmSqlServerTransparentDataEncryptionCertificate [-ResourceGroupName] <String> [-ServerName] <String>
+ [-PrivateBlob] <String> [-Password] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Add-AzureRmSqlManagedInstanceTransparentDataEncryptionCertificate adds a Transparent Data Encryption Certificate for the given SQL Server instance
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\>     Add-AzureRmSqlServerTransparentDataEncryptionCertificate -ResourceGroupName "YourResourceGroupName" -ServerName "YourServerName" -PrivateBlob "MIIJ+QIBAzCCCbUGCSqGSIb3DQEHAaCCCaYEggmiMIIJnjCCBhcGCSqGSIb3Dasdsadasd" -Password "CertificatePassword"
+```
+
```

Comment:
Please add piping scenarios examples.

Reply:
And you may want to add test cases for piping scenario.

Reply:
Added piping scenario examples. Test cases exist for both piping and resourceId